site/pt/blog/new-in-devtools-111/index.md

```diff
@@ -63,48 +72,61 @@ draft: true
 <!-- - Both pause exception options moved to the **Breakpoints** pane and labeled with text to make it more self-explanatory. -->
+- Ambas as opções de exceção de pausa foram movidas para o painel **Breakpoints** e rotuladas com texto para torná-lo mais autoexplicativo.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KADuAqVdrxxQDg5AYYeA.png", alt="Pause exception options.", width="800", height="518" %}
 
 <!-- - Breakpoints are grouped by file, ordered by line or column numbers, and are collapsible. -->
+- Os breakpoints são agrupados por arquivo, ordenados por números de linha ou coluna e são recolhíveis.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/AXJ7IL5aJ6gwbZmnoH39.png", alt="Group breakpoints by file.", width="800", height="454" %}
 
 <!-- - There are new options to deactivate, remove, and edit breakpoints when hovering over a breakpoint or file. -->
+- Existem novas opções para desativar, remover e editar breakpoints ao passar o mouse sobre um deles ou arquivo.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/vAQ4YuWGuXYLxXQXt0HG.png", alt="New options to deactivate breakpoints.", width="800", height="496" %}
 
 <!-- - Click the edit breakpoint button to open the breakpoint editor. From here, you can enter the breakpoint condition or switch to a logpoint. -->
+- Clique no botão editar breakpoint para abrir o editor. A partir daqui, você pode inserir a condição do breakpoint ou alternar para um ponto de log.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/SrgbxxF8U3s9yzDPV25q.png", alt="Breakpoint edit dialog.", width="800", height="697" %}
 
 <!-- See [JavaScript debugging reference](/docs/devtools/javascript/reference/) to learn how to debug with DevTools. -->
+Consulte [Referência de debugging para JavaScript](/docs/devtools/javascript/reference/) para aprender como depurar com o DevTools.
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/48b648b65cd05071d1950e50d0b529ff20294780 #} 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/b6c8a19b3922ed95818b5751f1b6548724ff868c #}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/0b955a3533292790168431db4e54906d4b1fa64a #} 
-Chromium issues: [1407586](https://crbug.com/1407586), [1402891](https://crbug.com/1402891), [1402893](https://crbug.com/1402893)
+Issues relacionadas: [1407586](https://crbug.com/1407586), [1402891](https://crbug.com/1402891), [1402893](https://crbug.com/1402893)
 
 <!-- ## Customizable Recorder shortcuts {: #recorder } -->
+## Atalhos customizáveis no Gravador {: #recorder }
 
 <!-- Use keyboard shortcuts to record and replay user flows quicker. -->
+Use atalhos de teclado para gravar e reproduzir os fluxos do usuário mais rapidamente.
 
 <!-- The **Recorder** introduces a few convenient keyboard shortcuts for faster recording and replaying of user flows.  -->
+O **Gravador** apresenta alguns atalhos de teclado convenientes para gravação e reprodução mais rápidas de fluxos de usuários.
```

Comment:
I see we sometimes reference the DevTools panels or features by the name in english and other times by the translated name. I'd recommend settling on one or the other and keeping consistent across the article.

Reply:
I think we are using the panel names in all the cases, are we using it inconsistently?